src/libraries/Common/tests/System/Net/Http/TestHelper.cs

```diff
@@ -90,7 +90,7 @@ public static byte[] ComputeMD5Hash(byte[] data)
 
         public static Task WhenAllCompletedOrAnyFailed(params Task[] tasks)
         {
-            return TaskTimeoutExtensions.WhenAllOrAnyFailed(tasks, PlatformDetection.IsArmProcess || PlatformDetection.IsArm64Process ? PassingTestTimeoutMilliseconds * 5 : PassingTestTimeoutMilliseconds);
+            return TaskTimeoutExtensions.WhenAllOrAnyFailed(tasks, PlatformDetection.IsArmProcess || PlatformDetection.IsArm64Process || PlatformDetection.IsRiscv64Process ? PassingTestTimeoutMilliseconds * 5 : PassingTestTimeoutMilliseconds);
```

Comment:
Are there details on this?

Reply:
No. Seems it is no needed here cause this not included in any launched testcase. If it would be needed in the future I will make another patch. Thanks.